go.mod

```diff
@@ -4,10 +4,8 @@ go 1.13
 
 require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.2.0
-	github.com/go-logr/logr v0.1.0
 	github.com/gogo/protobuf v1.3.1
-	github.com/onsi/ginkgo v1.13.0
-	github.com/onsi/gomega v1.10.1
+	github.com/onsi/ginkgo v1.13.0 // indirect
```

Comment:
Where are we using ginkgo?

Reply:
I think because of the controller-runtime dependency:

```shell
$ go mod why github.com/onsi/ginkgo                    
# github.com/onsi/ginkgo
sigs.k8s.io/service-apis/tools
sigs.k8s.io/controller-runtime/pkg/scheme
sigs.k8s.io/controller-runtime/pkg/scheme.test
github.com/onsi/ginkgo
```